diff --git a/base/feature_list.h b/base/feature_list.h
index 33adaca8ab5288a8d7c921c1b0bd3475d2b4808f..2f0a6f71fcee987edbd9e8553ca35a5767cbe62a 100644
--- a/base/feature_list.h
+++ b/base/feature_list.h
@@ -97,6 +97,11 @@ class BASE_EXPORT FeatureList {
   FeatureList& operator=(const FeatureList&) = delete;
   ~FeatureList();
 
+  bool IsFeatureOverridden_ChromiumImpl(std::string_view feature_name) const;
+  static std::optional<bool> GetStateIfOverridden_ChromiumImpl(
+      const Feature& feature);
+  static FeatureState GetCompileTimeFeatureState(const Feature& feature);
+
   // Used by common test fixture classes to prevent abuse of ScopedFeatureList
   // after multiple threads have started.
   class BASE_EXPORT ScopedDisallowOverrides {
