diff --git a/base/BUILD.gn b/base/BUILD.gn
index 81c78f51c9c861124f686ab50efd78b461964918..fb4fe5f41227ea6793e9ea08d4f56a9f102f15ab 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5085,6 +5085,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5759,6 +5760,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
