diff --git a/chrome/browser/ui/page_action/page_action_pass_key.h b/chrome/browser/ui/page_action/page_action_pass_key.h
index ce58aff6558ab43454329283011080b86e9c6080..fe501573294c5b7658d50134c8b86715f1a2ae62 100644
--- a/chrome/browser/ui/page_action/page_action_pass_key.h
+++ b/chrome/browser/ui/page_action/page_action_pass_key.h
@@ -28,6 +28,7 @@ class PageActionPassKey {
   static PageActionPassKey PassKeyForTesting() { return PageActionPassKey(); }
 
  private:
+  friend class page_actions::chromium_impl::PageActionControllerImpl;
   friend class PageActionController;
   friend class PageActionControllerImpl;
   friend class PageActionView;
