diff --git a/chrome/browser/ui/browser_command_controller.h b/chrome/browser/ui/browser_command_controller.h
index a19c11d49091138fcf421b71ad7c19f58accccad..dee8febc5ebcaf5154d8f627e63b8a999742d0b1 100644
--- a/chrome/browser/ui/browser_command_controller.h
+++ b/chrome/browser/ui/browser_command_controller.h
@@ -126,6 +126,7 @@ class BrowserCommandController : public CommandUpdater,
                                     base::TimeTicks time_stamp) override;
 
  private:
+  friend class BraveBrowserCommandController;
 #if BUILDFLAG(IS_CHROMEOS)
   friend class BrowserCommandControllerBrowserTestLockedFullscreen;
 #endif
@@ -191,7 +192,7 @@ class BrowserCommandController : public CommandUpdater,
 
   // Update commands whose state depends on the type of fullscreen mode the
   // window is in.
-  void UpdateCommandsForFullscreenMode();
+  virtual void UpdateCommandsForFullscreenMode();
 
   // Update commands whose state depends on whether they're available to hosted
   // app windows.
