diff --git a/chrome/browser/ui/browser_window/internal/browser_window_features.cc b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
index b2044c126757e529a82bfa5cab67872bd10abc3c..98438d474b7f1130fb319ef50d20ed9de01fa27c 100644
--- a/chrome/browser/ui/browser_window/internal/browser_window_features.cc
+++ b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
@@ -303,7 +303,7 @@ void BrowserWindowFeatures::Init(BrowserWindowInterface* browser) {
           *browser, fullscreen_controller_.get(), app_browser_controller_.get(),
           browser->GetType(),
           BrowserInitState::From(browser)->create_params().trusted_source,
-          browser->GetUnownedUserDataHost());
+          browser->GetUnownedUserDataHost(), *browser);
 
   side_panel_registry_ =
       GetUserDataFactory().CreateInstance<SidePanelRegistry>(*browser, browser);
