diff --git a/chrome/browser/ui/views/page_info/page_info_view_factory.cc b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
index 5da04577e49dab88820a3d04d1b61dd89f3c2c0c..a224a35ed22fc5732b1f12c15c29869f98967074 100644
--- a/chrome/browser/ui/views/page_info/page_info_view_factory.cc
+++ b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
@@ -645,6 +645,9 @@ const ui::ImageModel PageInfoViewFactory::GetPermissionIcon(
 
   icon = &gfx::VectorIcon::EmptyIcon();
   switch (permission.type) {
+    case ContentSettingsType::AUTOPLAY:
+      icon = &kAutoplayStatusIcon;
+      break;
     case ContentSettingsType::COOKIES:
       icon =
           &(features::IsRoundedIconsEnabled() ? vector_icons::kDatabaseIcon
