diff --git a/chrome/browser/actor/site_policy.cc b/chrome/browser/actor/site_policy.cc
index d243903901b715638b12aca0cf618bcfb26acb65..5e6bf8ffc55443ccdb457adb4061d043ff1c3292 100644
--- a/chrome/browser/actor/site_policy.cc
+++ b/chrome/browser/actor/site_policy.cc
@@ -117,6 +117,7 @@ void MayActOnUrlInternal(const GURL& url,
     return;
   }
 
+  BRAVE_MAY_ACT_ON_URL_INTERNAL
   if (IsActorSafetyCheckDisabled()) {
     decision_wrapper->Accept();
     return;
