diff --git a/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h b/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
index 54ff1d613d36187021834e1322dc34b74fc14efe..5d8a1313a902de773481e1f1ad2d827c610637fc 100644
--- a/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
+++ b/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
@@ -31,6 +31,7 @@ class BrowserWindowModalDialogDelegate
       BrowserWindowInterface* browser);
 
   // ChromeWebModalDialogManagerDelegate:
+  bool IsWebContentsVisible(content::WebContents* web_contents) override;
   void SetWebContentsBlocked(content::WebContents* web_contents,
                              bool blocked) override;
   web_modal::WebContentsModalDialogHost* GetWebContentsModalDialogHost(
