diff --git a/chrome/browser/browsing_data/BUILD.gn b/chrome/browser/browsing_data/BUILD.gn
index 5dbae7571a1c022669ba2bc93e771e71b9ee1fd1..0bde7ce935d521e014bf2d43e118fd024564ae2c 100644
--- a/chrome/browser/browsing_data/BUILD.gn
+++ b/chrome/browser/browsing_data/BUILD.gn
@@ -12,6 +12,7 @@ source_set("constants") {
     "//content/public/browser",
     "//extensions/buildflags",
   ]
+  public_deps = [ "//brave/components/ai_chat/core/common/buildflags" ]
 }
 
 source_set("browsing_data") {
