diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
index 24ace712aa61f9bc39e641c1093d15a507cb8bb5..aa287160bf1b2cc062779c2f5aa827ab99efa90d 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
@@ -16,6 +16,7 @@ import '../icons.html.js';
 import '../privacy_icons.html.js';
 import '../settings_page/settings_section.js';
 import '../settings_shared.css.js';
+import './personalization_options.js';
 import './privacy_guide/privacy_guide_dialog.js';
 
 import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js';
