diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 564bb8ddd244cd7ab79175fa256d3e4e6bd7a415..d52db9c9de5222178d26629abf95fd4404de68c2 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4780,6 +4780,7 @@ const FeatureEntry::FeatureVariation kDictationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
