diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
index e84bc8567f61e8bbf3046d6c9f156afb2700ac36..e4b151914b15dc079a10fcaaa3a3e80081dec907 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
@@ -42,6 +42,8 @@ class HorizontalTabStripRegionViewOld : public TabStripRegionView {
   HorizontalTabStripRegionViewOld& operator=(
       const HorizontalTabStripRegionViewOld&) = delete;
   ~HorizontalTabStripRegionViewOld() override;
+  views::Button* new_tab_button() { return new_tab_button_; }
+  TabStripComboButton* GetComboButton() { return combo_button_; }
 
   // Returns true if |point| falls within the window caption area of the
   // horizontal tab strip. Returns false if the point hits an interactive child
@@ -105,12 +107,16 @@ class HorizontalTabStripRegionViewOld : public TabStripRegionView {
   bool HasLeadingButtons() const;
 
  private:
+  friend class BraveVerticalTabStripRegionView;
+  friend class BraveTabStrip;
+  friend class BraveHorizontalTabStripRegionView;
+  FRIEND_TEST_ALL_PREFIXES(VerticalTabStripBrowserTest, MinHeight);
   // Updates the border padding for `new_tab_button_`.  This should be called
   // whenever any input of the computation of the border's sizing changes.
   void UpdateButtonBorders();
 
   // Updates the left and right margins for the tab strip.
-  void UpdateTabStripMargin();
+  virtual void UpdateTabStripMargin();
 
   // Gets called on `Layout` and adjusts the x-axis position of the `view` based
   // on `offset`. This should only used for views that show before tab strip.
