diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index ea969aab06af75c302a1b942c82fe31b80b0d226..98a4dcee50ce0c0dfadcdf30b06336eb25113b48 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -190,6 +190,7 @@ source_set("internal") {
       "//chrome/browser/ui:browser_select_file_dialog_controller",
     ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
