diff --git a/chrome/browser/ui/views/side_panel/side_panel.cc b/chrome/browser/ui/views/side_panel/side_panel.cc
index 0206817453ff485698f6a5926d42b925f439a19c..09de0cf073a10634dabfcdbcd6bee64f5faa2576 100644
--- a/chrome/browser/ui/views/side_panel/side_panel.cc
+++ b/chrome/browser/ui/views/side_panel/side_panel.cc
@@ -222,6 +222,7 @@ class ContentParentView : public views::View, public views::ViewObserver {
   }
 
   gfx::RoundedCornersF GetRoundedCorners() {
+    return brave::GetPanelContentsRoundedCorners(browser_view_);
     ChromeDistanceMetric corner_radius =
         ChromeDistanceMetric::DISTANCE_SIDE_PANEL_CONTENT_RADIUS;
     return GetLayoutProvider()
