diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 19eaec22285aa82b7c2c62ff3f0a7d3414704098..d333211dac83a45f2ccacb305da0fb6a6a3904d6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -701,7 +701,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             var modalDialogManager = getModalDialogManager();
             assert modalDialogManager != null;
             SnackbarManager snackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             windowAndroid,
@@ -3293,6 +3293,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
      */
     public void onExitVr() {}
 
+    public void onTabStateInitializedHandler() {}
     private void clearToolbarResourceCache() {
         View v = findViewById(R.id.control_container);
         try {
