diff --git a/chrome/browser/ui/startup/BUILD.gn b/chrome/browser/ui/startup/BUILD.gn
index 07de7a97892eade36ac8d42a72c67cbcba5c12a8..c79dab4b40e973e61afa232c26df2b6c75560114 100644
--- a/chrome/browser/ui/startup/BUILD.gn
+++ b/chrome/browser/ui/startup/BUILD.gn
@@ -28,6 +28,7 @@ source_set("startup_tab") {
   ]
 
   public_deps = [ "//url" ]
+  import("//brave/browser/ui/startup/sources.gni") public_deps += brave_browser_ui_startup_public_deps
 }
 
 source_set("startup") {
