diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index 6a474328b9860a84e78b214a77133caf7fcca750..8d2af594ca4edfc92aa63244c491fb17fad740fd 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -84,6 +84,7 @@
 #include "ui/base/ui_base_features.h"
 #endif
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -102,7 +103,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
