diff --git a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
index 535cb0dd2e8e87e68dc64cbf8f68f4e2f7e8f844..c95de9a97e11b69fe61a96f321aa38d34ae3ebc2 100644
--- a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
+++ b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
@@ -265,6 +265,8 @@ void TabStripView::OnChildMoved(TabCollectionNode* moved_node) {
 
   if (is_active_tab || has_focus) {
     switch (moved_node->type()) {
+      case TabCollectionNode::Type::TREE_NODE:
+        break;
       case TabCollectionNode::Type::TAB: {
         const tabs::TabInterface* tab =
             std::get<const tabs::TabInterface*>(moved_node->GetNodeData());
