diff --git a/components/sync/base/data_type.h b/components/sync/base/data_type.h
index a2f56d4062eaf0bdf718ca6532be42ccb53d7ba0..0d21b8a098c8ce5955f3a2ede756e83a9281e4ee 100644
--- a/components/sync/base/data_type.h
+++ b/components/sync/base/data_type.h
@@ -207,7 +207,8 @@ enum DataType {
   // Encrypted tab context item.
   ENCRYPTED_TAB_CONTEXT_ITEM,
 
-  LAST_USER_DATA_TYPE = ENCRYPTED_TAB_CONTEXT_ITEM,
+  AI_CHAT_CONVERSATION,
+  LAST_USER_DATA_TYPE = AI_CHAT_CONVERSATION,
 
   // ---- Control Types ----
   // An object representing a set of Nigori keys.
@@ -319,7 +320,8 @@ enum class DataTypeForHistograms {
   kThemesAndroid = 80,
   kEncryptedTabContextContainer = 81,
   kEncryptedTabContextItem = 82,
-  kMaxValue = kEncryptedTabContextItem,
+  kAIChatConversation = 82 + 1,
+  kMaxValue = kAIChatConversation,
 };
 // LINT.ThenChange(/tools/metrics/histograms/metadata/sync/enums.xml:SyncDataTypes)
 
