diff --git a/components/autofill/core/browser/ui/autofill_external_delegate.cc b/components/autofill/core/browser/ui/autofill_external_delegate.cc
index f3e996d79274d2f249de4d44d7e9e61d7fab5877..65c8e73ea4bb5b1f50ff268318bdfdb7e65e1045 100644
--- a/components/autofill/core/browser/ui/autofill_external_delegate.cc
+++ b/components/autofill/core/browser/ui/autofill_external_delegate.cc
@@ -461,6 +461,7 @@ void AutofillExternalDelegate::AttemptToDisplayAutofillSuggestions(
   // updated to match.
   InsertDataListValues(last_query_.field_datalist_options, suggestions);
 
+  BRAVE_AUTOFILL_EXTERNAL_DELEGATE_ATTEMPT_TO_DISPLAY_AUTOFILL_SUGGESTIONS
   // TODO(crbug.com/362630793): Try to eliminate this state. The controller
   // should be the one that knows about what suggestions were shown and passes
   // it on, not AED.
