diff --git a/components/sync/protocol/entity_specifics.proto b/components/sync/protocol/entity_specifics.proto
index fa8505c8a87593d36a4733448c0fcca7fa94c0d5..8f44003c957ba86d1320c00c9d09745b2d154187 100644
--- a/components/sync/protocol/entity_specifics.proto
+++ b/components/sync/protocol/entity_specifics.proto
@@ -11,6 +11,7 @@ option optimize_for = LITE_RUNTIME;
 
 package sync_pb;
 
+import "brave/components/sync/protocol/ai_chat_specifics.proto";
 import "components/sync/protocol/ai_thread_specifics.proto";
 import "components/sync/protocol/contextual_task_specifics.proto";
 import "components/sync/protocol/encryption.proto";
@@ -179,6 +180,7 @@ message EntitySpecifics {
     EncryptedTabContextContainerSpecifics encrypted_tab_context_container =
         1656053;
     EncryptedTabContextItemSpecifics encrypted_tab_context_item = 1661069;
+    AIChatConversationSpecifics ai_chat_conversation = 10000001;
     // When adding a new type, follow the docs below and keep this comment as
     // the last entry.
     // https://www.chromium.org/developers/design-documents/sync/integration-checklist/
