diff --git a/components/sync/engine/cycle/data_type_tracker.cc b/components/sync/engine/cycle/data_type_tracker.cc
index b45c103c6339fad69de60f77c2768616e4808f7e..1d11942803f1098b971d6cb7e36ba773586cadf2 100644
--- a/components/sync/engine/cycle/data_type_tracker.cc
+++ b/components/sync/engine/cycle/data_type_tracker.cc
@@ -121,6 +121,7 @@ base::TimeDelta GetDefaultLocalChangeNudgeDelay(DataType data_type) {
     case SKILL:
     case GEMINI_THREAD:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return kMediumLocalChangeNudgeDelay;
     case UNSPECIFIED:
       NOTREACHED();
@@ -201,6 +202,7 @@ bool CanGetCommitsFromExtensions(DataType data_type) {
     case SKILL:
     case GEMINI_THREAD:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return false;
     case UNSPECIFIED:
       NOTREACHED();
