diff --git a/content/browser/renderer_host/clipboard_host_impl.cc b/content/browser/renderer_host/clipboard_host_impl.cc
index 9738c912b370b01e4bee596c5da3c7e41e65ff37..94ffcb077854207cfded3a4e329fcb19bd55df33 100644
--- a/content/browser/renderer_host/clipboard_host_impl.cc
+++ b/content/browser/renderer_host/clipboard_host_impl.cc
@@ -610,6 +610,7 @@ void ClipboardHostImpl::WriteText(const std::u16string& text) {
   ClipboardPasteData data;
   data.text = text;
   ++pending_writes_;
+  BRAVE_CLIPBOARD_HOST_IMPL_SANITIZE
   GetContentClient()->browser()->IsClipboardCopyAllowedByPolicy(
       CreateClipboardEndpoint(render_frame_host()),
       {
