diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index b58c131d355763bf0c118d1d87a918785ecf91b8..0dacd32839d5675af17431ce97d356ff54e17347 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -64,6 +64,7 @@ source_set("common") {
   # must not link to this because it will duplicate the code in the component
   # build.
   visibility = [ "//content/*" ]
+  visibility += [ "//brave/content/*" ]
 
   sources = [
     "background_fetch/background_fetch_types.cc",
