diff --git a/content/browser/renderer_host/navigation_request.cc b/content/browser/renderer_host/navigation_request.cc
index dd571d69bb6e7ef291e47dcdcc5cb6c7d46b84b3..4037059cd5dbd38b0907fc52d41d3511934ea02e 100644
--- a/content/browser/renderer_host/navigation_request.cc
+++ b/content/browser/renderer_host/navigation_request.cc
@@ -4065,6 +4065,9 @@ void NavigationRequest::OnRequestRedirected(
   common_params_->url = redirect_info.new_url;
   common_params_->method = redirect_info.new_method;
   common_params_->referrer->url = GURL(redirect_info.new_referrer);
+  GetContentClient()->browser()->MaybeHideReferrer(
+      browser_context, common_params_->url,
+      GetTopDocumentGURL(frame_tree_node_), &common_params_->referrer);
   common_params_->referrer = Referrer::SanitizeForRequest(
       common_params_->url, *common_params_->referrer);
 
@@ -6034,6 +6037,10 @@ void NavigationRequest::OnStartChecksComplete(
   // TODO(clamy): Avoid cloning the navigation params and create the
   // ResourceRequest directly here.
 
+  GetContentClient()->browser()->MaybeHideReferrer(
+      frame_tree_node_->navigator().controller().GetBrowserContext(),
+      common_params_->url, GetTopDocumentGURL(frame_tree_node_),
+      &common_params_->referrer);
   auto loader_type = NavigationURLLoader::LoaderType::kRegular;
   network::mojom::URLResponseHeadPtr cached_response_head = nullptr;
   if (IsServedFromBackForwardCache()) {
