diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 203f8cf75131a692a771ddd6053681b2261f5fc5..95781746a5a6bf862da1a2df1d5132d4526d7c46 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3769,6 +3769,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
