diff --git a/DEPS b/DEPS
index 88c50827a5823e164c97a40fe82ff480f70ab85a..2cbc5da35549287343c21e348d7f597cd207aacc 100644
--- a/DEPS
+++ b/DEPS
@@ -4438,7 +4438,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
