diff --git a/ios/web/web_state/ui/crw_web_controller.mm b/ios/web/web_state/ui/crw_web_controller.mm
index 97f38d084969fb5dd24229cf62bce736339e0b4a..4d84dd0b133152521ef06dcc7b8a2bde8b356d47 100644
--- a/ios/web/web_state/ui/crw_web_controller.mm
+++ b/ios/web/web_state/ui/crw_web_controller.mm
@@ -338,11 +338,11 @@ JavaScriptCompletionBlock WrapCompletionBlock(
       web::AnnotationsTextManager::CreateForWebState(_webStateImpl);
     }
 
-    _navigationHandler = [[CRWWKNavigationHandler alloc] initWithDelegate:self];
+    _navigationHandler = [[BraveCRWWKNavigationHandler alloc] initWithDelegate:self];
 
     _jsNavigationHandler = [[CRWJSNavigationHandler alloc] init];
 
-    _UIHandler = [[CRWWKUIHandler alloc] init];
+    _UIHandler = [[BraveCRWWKUIHandler alloc] init];
     _UIHandler.delegate = self;
 
     _webViewNavigationObserver = [[CRWWebViewNavigationObserver alloc] init];
