diff --git a/net/BUILD.gn b/net/BUILD.gn
index 269d851fe8af4e329d96d23dcea6276bfedce7ac..cec690534c92e2dc29c11a60eb2983b1faa3fd8a 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1291,6 +1291,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
