diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index b9f53668cd02b7f247f68718a7dd4a956d6e6065..51033617cc9e1a675fdd4ad42b7919140e13d6f1 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -784,6 +784,7 @@ void URLRequestHttpJob::AddExtraHeaders() {
       request()->context()->enable_brotli(),
       request()->context()->enable_zstd());
 
+  BRAVE_URL_REQUEST_HTTP_JOB_ADD_EXTRA_HEADERS
   if (http_user_agent_settings_) {
     // Only add default Accept-Language if the request didn't have it
     // specified.
