diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 7f03633233ee6d252dc807ffe06446a3281800bf..d2891176288d574fce35e46bd4a532e876bad673 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -397,6 +397,7 @@ component("network_service") {
     "//third_party/zstd:decompress",
     "//url",
   ]
+  import("//brave/services/network/sources.gni") deps += brave_services_network_deps
 
   if (is_linux || is_chromeos) {
     deps += [
