diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index a918b05fa880884ae8c59c886ddfca9711969982..c291ae7d79f2b70ba6a6a28b0594a8c8cf5405e9 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -379,6 +379,8 @@ if (!use_litert_tflite) {
 
     visibility = [
       ":tflite_benchmark",
+      "//brave/browser/history_embeddings/*",
+      "//brave/services/passage_embeddings/*",
       "//components/optimization_guide/internal/*",
       "//modules/*",
       "//services/webnn/*",
