diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index 25e54b1244522bd0ee7d330a7a86ff26d524e5f6..d0ea9a622de50d8c58556874a74323f75d6b17aa 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -7607,6 +7607,7 @@ KURL Document::CompleteURLWithOverride(
   if (result.IsEmpty()) {
     result = Encoding().IsValid() ? KURL(base_url_override, url, Encoding())
                                   : KURL(base_url_override, url);
+    result = RewriteBraveToChrome(result);
     url_cache_.Put(base_url_override, url, result);
   }
   // If the conditions are met for
