diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index 18d1c5391c0c80886782546c2687f9819a99c7d1..db0329929001a5065e52af859ac1e1b2ec68d68b 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -103,6 +103,13 @@ class CORE_EXPORT HTMLCanvasElement final
 
   explicit HTMLCanvasElement(Document&);
   ~HTMLCanvasElement() override;
+  String toDataURL(ScriptState* script_state, const String& mime_type,
+                   const ScriptValue& quality_argument,
+                   ExceptionState& exception_state) const;
+  String toDataURL(ScriptState* script_state, const String& mime_type,
+                   ExceptionState& exception_state) const {
+    return toDataURL(script_state, mime_type, ScriptValue(), exception_state);
+  }
 
   ElementType GetElementType() const final {
     return ElementType::kHTMLCanvasElement;
@@ -389,6 +396,7 @@ class CORE_EXPORT HTMLCanvasElement final
   void RemovedFrom(ContainerNode& insertion_point) override;
 
  private:
+  mutable UntracedMember<ExecutionContext> scoped_execution_context_;
   void Dispose();
 
   void ColorSchemeMayHaveChanged();
