diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 538b2cf02ae22af4f211735d59035e7b543c3218..7f3b19ca38d34f1ce9ffbd8e08437a5800d40156 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -121,6 +121,7 @@ generate_grd("build_grd") {
     public_deps += [ ":build_cros_styles_grdp" ]
     grdp_files += [ "$target_gen_dir/cros_styles_resources.grdp" ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos) {
