diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index dedde497596ba46d8a238ce5c01f423cb9757721..526714a4bfb25d64ad33016eb1393c6e7bf01b6b 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1691,6 +1691,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return `${host}:${port}`;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 export type RequestConditionsSetting = {
   url: string,
